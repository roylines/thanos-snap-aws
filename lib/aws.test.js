const aws = require('./aws');
const AWS = require('aws-sdk');

jest.mock('aws-sdk');

describe('aws', () => {
  describe('ec2', () => {
    it('can call describeInstances', async () => {
      const response = await aws.ec2.describeInstances('REQUEST');
      expect(response).toMatchSnapshot();
      expect(AWS.EC2().describeInstances.mock.calls).toMatchSnapshot();
    });
  });
});
