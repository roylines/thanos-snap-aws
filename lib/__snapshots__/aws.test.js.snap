// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aws ec2 can call describeInstances 1`] = `"describeInstancesResponse"`;

exports[`aws ec2 can call describeInstances 2`] = `
Array [
  Array [
    "REQUEST",
  ],
]
`;
